.github/workflows/push_rockspec.yml

```diff
@@ -0,0 +1,41 @@
+name: Push rockspec
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+env:
+  ROCK_NAME: "metrics"
+
+jobs:
+  push-scm-rockspec:
```

Comment:
`push scm rockspec` could start before tests

Should we prevent pushing rockspecs and artefacts when test are failing?